package.json

```diff
@@ -443,7 +443,7 @@
     "@elastic/eslint-plugin-eui": "0.0.2",
     "@elastic/github-checks-reporter": "0.0.20b3",
     "@elastic/makelogs": "^6.1.1",
-    "@elastic/synthetics": "^1.0.0-beta.12",
+    "@elastic/synthetics": "~1.0.0-beta.13",
```

Comment:
Changed this to `~` because `^` bumps this up to `1.3.0`, which is incompatible with the Node.js runtime that we use for 7.17.